src/components/originalSampleProcessing/ProcessingSuccess.tsx

```diff
@@ -11,13 +11,20 @@ import { Column } from 'react-table';
 import Heading from '../Heading';
 import { ModalBody } from '../Modal';
 import BlueButton from '../buttons/BlueButton';
+import { reload } from '../../lib/sdk';
+import WhiteButton from '../buttons/WhiteButton';
+import { createSessionStorageForLabwareAwaiting } from '../../types/stan';
 interface ProcessingSuccessProps {
   labware: LabwareFieldsFragment[];
   columns: Column<LabwareFieldsFragment>[];
   successMessage: string;
 }
 
 const ProcessingSuccess: React.FC<ProcessingSuccessProps> = ({ labware, columns, successMessage }) => {
```

Comment:
We have a very similar component called OperationCompleteModal in src/compoents/modal which is extendable as well as it is hoc. I would probably recommend to use it than to creating a new one, unless there is specific reasons.